index.js

```diff
@@ -0,0 +1,241 @@
+'use strict'
+
+const { randomUUID } = require('node:crypto')
+const { major: semverMajor } = require('semver')
+const { RefResolver } = require('json-schema-ref-resolver')
+const { compareObjectKeys } = require('./lib/utils.js')
+
+const HTTP_METHODS = ['get', 'put', 'post', 'delete', 'options', 'head', 'patch', 'trace']
```

Comment:
It's likely better to use Node.js core list of methods, or maybe this is limited for a reason?

Reply:
this is a list from openapi specification
https://swagger.io/specification/#path-item-object